packages/design-system/src/components/Tooltip/TooltipContent.js

```diff
@@ -23,6 +25,8 @@ const TooltipContent = ({
     }),
     className
   );
+  
+  noticeAboutDeprecation('[Design System] deprecated component - Tooltip, please use CSSTooltip or PopperTooltip instead');
```

Comment:
maybe this `[Design System]` prefix could be added by the `noticeAboutDeprecation` function itself? So it will be guaranteed that all depreciation warnings will have it.

Reply:
yup, a very good idea! :)

Reply:
done ;)

Reply:
done 🌮